include/tbl/vsl_tags.h

```diff
@@ -117,9 +117,22 @@ SLTM(BackendOpen, 0, "Backend connection opened",
 	"\n"
 )
 
-SLTM(BackendReuse, 0, "Backend connection put up for reuse",
-	"Logged when a backend connection is put up for reuse by a later"
-	" connection.\n\n"
+SLTM(BackendReuse, 0, "Backend connection reused",
+	"Logged when an existing backend connection is reused.\n\n"
+	"The format is::\n\n"
+	"\t%d %s %s %s %s %s\n"
+	"\t|  |  |  |  |  |\n"
+	"\t|  |  |  |  |  +- Local port\n"
+	"\t|  |  |  |  +---- Local address\n"
+	"\t|  |  |  +------- Remote port\n"
+	"\t|  |  +---------- Remote address\n"
+	"\t|  +------------- Backend display name\n"
+	"\t+---------------- Connection file descriptor\n"
+	"\n"
+)
+
+SLTM(BackendRecycle, 0, "Backend connection recycled",
```

Comment:
new tags need to be added at the end of the file to preserve binary compatibility of vsl files